action.yml

```diff
@@ -1,9 +1,9 @@
-name: 'Setup Dotnet environment'
-description: 'Setup a Dotnet environment and add it to the PATH, additionally providing proxy support'
+name: 'Setup .NET SDK'
```

Comment:
Could we make it more clear that it is dotnet core? There were some questions about 4.5.* not installing in a previous issue.

Reply:
I think the usual name used is `.NET Core SDK`, but our team's PM @kathleendollard probably has opinions here.

Reply:
I think we can set up the framework SDKs as well, at least on Windows.

Reply:
Does 4.5 *not work*?  It's documented.

Reply:
It should not be documented and has been removed from the README in master. This is only for installing the .NET Core SDK.

Reply:
Great, I updated it to say ".NET Core SDK".